src/modules/commentCounter.js

```diff
@@ -0,0 +1,12 @@
+import { getComment } from './involvementApi.js';
+
+const countComment = (array) => array.length;
+
+const commentCounter = (IdMeal) => {
+  const commentCounterEl = document.getElementById('commentCounter');
+  const CommentData = getComment(IdMeal);
+  CommentData.then((value) => {
+    commentCounterEl.innerHTML = `${value.length}`;
+  });
```

Comment:
Thank you @EricMbouwe , but are this necessary to get this project approved, as I think this will affect most of our codes which may make us miss the deadline